js/src/views/Address/address.js

```diff
@@ -153,7 +153,7 @@ class Address extends Component {
       <Button
         key='delete'
         icon={ <ActionDelete /> }
-        label='forget address'
+        label='forget'
```

Comment:
I have an i18n branch going, starting my way up from the bottom of the views, will get to that in there. Not doing in here, minor tweaks before release, nothing more.